pyproject.toml

```
@@ -0,0 +1,58 @@
+[build-system]
+requires = ["setuptools>=68.0", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[tool.setuptools]
+platforms = ["Linux x86, x86-64"]
+
+[tool.setuptools.packages.find]
+include = ["sopel_slap", "sopel_slap.*"]
+namespaces = false
+
+[project]
+name = "sopel-slap"
+version = "0.2.0.dev0"
+description = "Plugin for Sopel that lets users slap each other in fun ways"
+keywords = [
+  "sopel",
+  "plugin",
+  "slap",
+  "pranks",
+  "bot",
+  "irc",
+]
+maintainers = [
+  { name="dgw" },
+]
+authors = [
+  { name="dgw" },
+  { name="Michael Yanovich" },
+]
+
+readme = "README.md"
+license = { text="EFL-2.0" }
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Intended Audience :: Developers",
+    "Intended Audience :: System Administrators",
+    "License :: Eiffel Forum License (EFL)",
+    "License :: OSI Approved :: Eiffel Forum License",
+    "Operating System :: POSIX :: Linux",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Topic :: Communications :: Chat :: Internet Relay Chat",
+]
+requires-python = ">=3.8, <4"
+dependencies = [
+    "sopel>=7",
```

I thought I'd suggested leaving only Sopel 8 support, but maybe not. Regardless, since this plugin imports `sopel.plugin` it must require 7.1+ at minimum.


```suggestion
    "sopel>=7.1",
```

---

Hmm, to heck with this; the mistake is a holdover from when I initially put the plugin together. I'll fix it during release.